samples/generic-base/monitoring/user.env

```diff
@@ -56,6 +56,13 @@
 # LOGGING_DASH=true
 # USER_DASH=true
 
+# OpenShift-specific options
+# Used when OPENSHIFT_PATH_ROUTES=true
+# Fully-qualified route hostname used for Grafana
+# OPENSHIFT_ROUTE_HOST_GRAFANA
```

Comment:
Should we include a sample value here?  Or, iff that's not possible, an obvious non-sense value?  As it is now, there is just a variable name with no assignment operator or value...not at all clear that this is a value that should/can be changed.

Reply:
Updated